map/mwm_url.cpp

```diff
@@ -22,9 +22,53 @@ namespace url_scheme
 
 namespace
 {
+string const kLatLon = "ll";
+string const kSourceLatLon = "sll";
+string const kDestLatLon = "dll";
+string const kZoomLevel = "z";
+string const kName = "n";
+string const kSourceName = "saddr";
+string const kDestName = "daddr";
+string const kId = "id";
+string const kStyle = "s";
+string const kBackUrl = "backurl";
+string const kVersion = "v";
+string const kAppName = "appname";
+string const kBalloonAction = "balloonaction";
+string const kRouteType = "type";
 
 static int const INVALID_LAT_VALUE = -1000;
 
+bool ParseLatLon(double & lat, double & lon, string const & key, string const & value)
+{
+  size_t const firstComma = value.find(',');
+  if (firstComma == string::npos)
+  {
+    LOG(LWARNING, ("Map API: no comma between lat and lon for 'll' key", key, value));
+    return false;
+  }
+
+  if (value.find(',', firstComma + 1) != string::npos)
```

Comment:
What's the point to find second comma? I guess to_double should fail when applied to wrong argument, right?